CHECKS

```diff
@@ -0,0 +1 @@
+/   Nothing to see here
```

Comment:
This is useful when deploying using Dokku, so it would be a good addition to the repo for me.

Reply:
Useful in what way? Is it possible to document the reason for this file in the file itself for future maintainer(s)?

Reply:
actually this should have something in it... *looks around for his file*